lib/raven/breadcrumbs/logger.rb

```diff
@@ -0,0 +1,79 @@
+require 'logger'
+
+module Raven
+  module BreadcrumbLogger
+    LEVELS = {
+      ::Logger::DEBUG => 'debug',
+      ::Logger::INFO  => 'info',
+      ::Logger::WARN  => 'warn',
+      ::Logger::ERROR => 'error',
+      ::Logger::FATAL => 'fatal'
+    }.freeze
+
+    EXC_FORMAT = /^([a-zA-Z0-9]+)\:\s(.*)$/
+
+    def self.parse_exception(message)
+      lines = message.split(/\n\s*/)
+      return nil unless lines.length > 2
+
+      match = lines[0].match(EXC_FORMAT)
+      return nil unless match
+
+      _, type, value = match.to_a
+      [type, value]
+    end
+
+    def add(severity, message = nil, progname = nil, &block)
```

Comment:
rather than reimplement Logger#add, we should use `super` or an aliased method. can fix after merge.